NextDom/nextdom-core
# issue 719 + Change autocomplete scenario + tweaks

- fix issue #719 
- Change autocomplete lib for scenario action & condition 
- delete jquery.sew lib (out of date) for use use jquery-ui
- Clean jquery-sew
- Add interlink bouton in double page (param & tools) like logs-view & logs-admin

To test :
- use interlink button in "double" pages (pages that you found in admin/param & admin/tools for exemple)
- use scenario page and scenario edition page
- try autocomplete action and autocomplete IF condition
- check scenario edition title (ID, state)
